### DISCUSSION TITLE
Get Selected Text

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#3741] [Q&A]
Hi @artf  Is there any way from where we can get the selected text on the selected component?
PFA. Thanks

<img width="1221" alt="Screenshot 2021-09-02 at 10 55 57 AM" src="https://user-images.githubusercontent.com/20657737/131786826-b0424efe-01cb-4703-bbb7-01342ba317e0.png">

**Comment by artf** [comment]
You can get the window from the selected component and then use the HTML Selection API, eg:
```js
const elDom = editor.getSelected().getEl();
const elWindow = elDom.ownerDocument.defaultView;
console.log('selected text:', elWindow.getSelection().toString());
```
